packages/backend/src/accounting/tigerbeetle/accounts.ts

```diff
@@ -39,7 +40,7 @@ export async function createAccounts(
       ledger: account.ledger,
       code: account.code,
       flags:
-        account.type === AccountType.Debit
+        account.type === TigerbeetleAccountType.Debit
```

Comment:
I wonder if we can remove `TigerbeetleAccountType` and instead use the `AccountType` from `accounting/service` and do a settlement or liquidity check here